.github/workflows/unittests.yml

```diff
@@ -26,11 +26,25 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v2
 
-      # Install OS specific dependencies
+      - name: Compilation cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.ccache
+          # We include the commit sha in the cache key, as new cache entries are
+          # only created if there is no existing entry for the key yet.
+          key: ${{ runner.os }}-ccache-${{ github.sha }}
+          # Restore any ccache cache entry, if none for
+          # ${{ runner.os }}-ccache-${{ github.sha }} exists
+          restore-keys: |
+            ${{ runner.os }}-ccache
+
+      # Install Linux specific dependencies
       - name: Install Linux dependencies
         if: matrix.os == 'ubuntu-latest'
         # TODO https://github.com/apache/incubator-mxnet/issues/18293
-        run: sudo apt-get install libopenblas-dev
+        run: |
+          sudo apt-get install -y libopenblas-dev ninja-build libedit-dev libxml2-dev
+          sudo bash -c "$(wget -O - https://apt.llvm.org/llvm.sh)"
```

Comment:
Unused?

Reply:
Do you mean llvm or `libedit-dev libxml2-dev`?

Reply:
For LLVM, it's used in TVM.

Reply:
May be it's too slow to ask Github actions to test TVM and I'm considering to remove the check.

Reply:
I've revised the test case to only test one model when it's on CPU.

Reply:
I've marked it as resolved. Feel free to update your concern.

Reply:
I mean that you may not need to add the llvm apt repo: `sudo bash -c "$(wget -O - https://apt.llvm.org/llvm.sh)"`

Reply:
> LLVM 4.0 or higher is needed for build with LLVM. Note that version of LLVM from default apt may lower than 4.0.

You don't need to add the upstream llvm repo. You can just use the llvm version available on Ubuntu itself. It may faster and less flaky (upstream website may be more likely to be unavailable at times).